src/model/sample.rs

```diff
@@ -324,11 +323,21 @@ impl Sample {
                     let end_pos = cigar.end_pos() as u32;
 
                     if pos <= start && end_pos >= start {
-                        let cigar = record.cigar();
-                        observations.push(
-                            self.read_observation(&record, &cigar, start, variant, chrom_seq)?
-                        );
-                        n_overlap += 1;
+                        if let Some( (prob_ref, prob_alt) ) = evidence::reads::prob_snv(record, &cigar, start, variant, chrom_seq)? {
```

Comment:
I initially did it that way, but figured that it complicates read_observations() quite a bit, having to pass on the possible `None` from prob_snv() and therefor artificially wrapping the observation from an indel in an `Option<>`, as well. So I think it is the cleaner solution this way, but if you are e.g. expecting similar reasons for skipping reads in indels, as we have to for SNVs when the variant position is deleted ('D') or ref skipped ('N'), I can quickly merge this back together to facilitate such future changes.